test/controllers/test-logout.js

```diff
@@ -77,21 +94,22 @@ describe('logout', function () {
 
   describe('when Accept header is set to application/json', function () {
     it('should respond with 200', function (done) {
-      var config = app.get('stormpathConfig');
       request(app)
         .post(config.web.logout.uri)
         .set('Accept', 'application/json')
-        .expect(200)
-        .end(done);
+        .expect(200).end(done);
     });
   });
 
   it('should follow the next param if present', function (done) {
-    var config = app.get('stormpathConfig');
     request(app)
       .post(config.web.logout.uri + '?next=/goodbye')
       .expect(302)
       .expect('Location', '/goodbye')
       .end(done);
   });
+
+  it('should call the postLogoutHandler', function () {
+    assert(postLogoutHandlerSpy.callCount, 3);
```

Comment:
Yeah, I just added that as a quick check. I'm going to add some more extensive tests. I'm currently working through the `preLoginHandler` and `preRegistrationHandler` tests, but those are very slow to write.